src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -278,10 +278,11 @@ object Task {
 
   def tasksById(tasks: Iterable[Task]): Map[Task.Id, Task] = tasks.iterator.map(task => task.taskId -> task).toMap
 
-  case class Id(idString: String) {
+  case class Id(idString: String) extends Comparable[Id] {
```

Comment:
Nice, I didn't really think of `Ordered`.

I think that it is always nice to be able to sort things, e.g. for reproducibility.